ui/apps/dashboard/src/app/(organization-active)/(dashboard)/env/[environmentSlug]/apps/sync-new/ManualSync.tsx

```diff
@@ -18,7 +18,7 @@ import { graphql } from '@/gql';
 import { pathCreator } from '@/utils/urls';
 import { useEnvironment } from '../../../../../../../components/Environments/environment-context';
 
-const SyncNewAppDocument = graphql(`
+export const SyncNewAppDocument = graphql(`
```

Comment:
Later the syncApp logic should be moved to a separate package to be used in multiple places